src/test/java/seedu/address/logic/parser/SortStudentCommandParserTest.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SORT_BY;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.SortStudentCommand;
+
+/**
+ * Contains unit tests for SortStudentCommandParser.
+ */
+public class SortStudentCommandParserTest {
+
+    private SortStudentCommandParser parser = new SortStudentCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, SortStudentCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsSortStudentCommand() {
```

Comment:
Since you have handled the edge case in the parser, it would be nice if you could also show different user inputs such as "Name", "NAME", "name" (etc) to show that we covered all cases

Reply:
Nevertheless the current test cases are more than sufficient! Good job 👍